src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx

```diff
@@ -114,6 +130,22 @@ function SearchPageSearchForm({
       return getTranslation(divisionData.name, router.locale);
     }
 
+    if (key === "ontologyTreeIds") {
+      if (ontologyTreeQuery.loading) {
+        return <SmallSpinner color="white" />;
+      }
```

Comment:
The intention was to use a white spinner so that it matches the colour of the text and cross icon that's used with the keywords on the search filter list.